test/integration/bad/new/stanc.expected

```diff
@@ -1179,15 +1179,16 @@ Syntax error in 'ill-formed-statement21.stan', line 1, column 28 to column 33, p
                                      ^
    -------------------------------------------------
 
-Expected ")" after "get_lp(".
+Ill-formed function application. Expect comma-separated list of expressions followed by ")" after "(".
   $ ../../../../../install/default/bin/stanc ill-formed-statement22.stan
 Syntax error in 'ill-formed-statement22.stan', line 1, column 26 to column 31, parsing error:
    -------------------------------------------------
      1:  transformed data { get_lp while
                                    ^
    -------------------------------------------------
 
-Expected "()" after "get_lp".
+Unexpected input after the conclusion of a valid expression.
+You may be missing a "," between expressions, an operator, or a terminating "}", ")", "]", or ";".
```

Comment:
Many of these files provoked syntax errors that were specific to the now-removed features. I'm not sure if they should be removed completely but I think just letting them fallthrough into generic "ill-formed expression" errors is not good.

Reply:
I went ahead and removed these now-junk tests